src/megatron/bridge/recipes/qwen_vl/qwen3_vl.py

```
@@ -23,9 +23,14 @@
 from typing import Optional, Union
 
 import torch
-from transformers import AutoTokenizer, Qwen3VLProcessor
+from transformers import AutoTokenizer
 from typing_extensions import TypedDict, Unpack
 
+try:
+    from transformers import Qwen3VLProcessor
+except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)